dto/src/main/java/org/jboss/pnc/rex/dto/ConfigurationDTO.java

```diff
@@ -33,5 +33,6 @@
 @ToString
 public class ConfigurationDTO {
 
-    public boolean passResultsOfDependencies = false;
+    private boolean passResultsOfDependencies = false;
+    private boolean skipCallback = false;
```

Comment:
could you rename this to fastStart or skipStartCallback?

Reply:
yup sure!

Reply:
The `skipStartCallback` naming won't work if I also include this for cancellation.